hyp3lib/simplify_shapefile.py

```diff
@@ -43,8 +41,7 @@ def wkt2shape(wkt,output_file):
 
 def simplify_shapefile(inshp,outshp):
     if not os.path.isfile(inshp):
-        logging.error("ERROR: File {} does not exist".format(inshp))
-        sys.exit(1)
+        raise FileNotFoundError(f"{inshp} does not exist")
```

Comment:
I agree drop `ERROR: `, but maybe keep `File `, i.e. `f"File {inshp} does not exist"`

Reply:
Because it's a `FileNotFoundError` it's pretty explicit it's a file/directory that doesn't exist:
![image](https://user-images.githubusercontent.com/7882693/84082326-2638e580-a98c-11ea-857c-9222c228d2c7.png)
 
Though, maybe it should be in quotes?